homeassistant/components/fujitsu_hvac/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""The Fujitsu HVAC (based on Ayla IOT) integration."""
+from __future__ import annotations
+
+from asyncio import timeout
+from contextlib import suppress
+
+from ayla_iot_unofficial import AylaAuthError, new_ayla_api
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+
+from .const import API, API_TIMEOUT, AYLA_APP_ID, AYLA_APP_SECRET, CONF_EUROPE, DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.CLIMATE]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Fujitsu HVAC (based on Ayla IOT) from a config entry."""
+    api = new_ayla_api(
+        entry.data[CONF_USERNAME],
+        entry.data[CONF_PASSWORD],
+        AYLA_APP_ID,
+        AYLA_APP_SECRET,
+        europe=entry.data[CONF_EUROPE],
+    )
+
+    try:
+        async with timeout(API_TIMEOUT):
+            await api.async_sign_in()
```

Comment:
Can we maybe move the timeouts to the lib? As they are kinda service specific (and it would clean up the code). Not a blocker, but would be preferred

Reply:
I'd rather not : this library is used by other integrations which might not expect this behaviour.

Reply:
But why are we adding a timeout then?